transit_odp/publish/tests/test_views.py

```diff
@@ -2045,3 +2052,71 @@ def test_require_attention_all_variations(
 
     assert response.status_code == 200
     assert len(response.context["view"].object_list) == 6
+    # One out of season seasonal service reduces in scope services to 8
+    assert response.context["services_require_attention_percentage"] == 75
+
+
+@pytest.mark.parametrize(
+    "pathname,pathargs,pathkwargs",
+    [
+        ("requires-attention", [], {"pk1": 1}),
+    ],
+)
```

Comment:
You don't need to parameterise it if you're only going to run it with one set of input data.